kedro/utils.py

```diff
@@ -160,6 +160,23 @@ def _has_rich_handler(logger: logging.Logger | None = None) -> bool:
     return any(isinstance(handler, RichHandler) for handler in logger.handlers)
 
 
+def _has_only_rich_handlers(logger: logging.Logger | None = None) -> bool:
```

Comment:
Since this method is very similar to `_has_rich_handlers`, could we just use this new method everywhere instead?